setup.py

```diff
@@ -54,17 +53,8 @@
     install_requires=install_requires,
     package_dir={"": "python"},
     packages=setuptools.find_namespace_packages(where="python"),
-    package_data={"": ["*.rst", "*.yaml", "*.xml", "*.jschema"]},
-    data_files=[
-        (os.path.join(data_files_path, "schema"), ["schema/amcs_status.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/apscs_status.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/command.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/lcs_status.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/lwscs_status.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/moncs_status.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/response.jschema"]),
-        (os.path.join(data_files_path, "schema"), ["schema/thcs_status.jschema"]),
-    ],
+    package_data={"": ["*.rst", "*.yaml", "*.xml"]},
```

Comment:
Please try to figure out why flake8 didn't complain about unused variable `data_files_path`.

Once you get that figured out, please delete lines 41-43 (which define tools_path, base_prefix, and data_files_path).

Reply:
I honestly don't know why Mypy doesn't complain but I could remove those lines and two unused imports after that and Mypy still doesn't complain.

Reply:
It's `flake8` I am worried about, not `mypy`. I fear `setup.cfg` may needlessly be excluding `setup.py` from flake8 checking.